tools/ci_build/build.py

```diff
@@ -445,7 +445,10 @@ def parse_arguments():
                         "See /docs/ONNX_Runtime_Format_Model_Usage.md for more information. "
                         "RTTI is automatically disabled in a minimal build. "
                         "To enable execution providers that compile kernels at runtime (e.g. NNAPI) pass 'extended' "
-                        "as a parameter. e.g. '--minimal_build extended'.")
+                        "as a parameter. e.g. '--minimal_build extended'. "
+                        "To enable support for custom operators pass 'custom_ops' as a parameter. "
+                        "e.g. '--minimal_build custom_ops'. This can be combined with an 'extended' build by passing "
+                        "'--minimal_build extended,custom_ops'")
```

Comment:
>extended,custom_ops [](start = 42, length = 19)

how about something like `add_argument("--minimal_build", nargs="*", default=None, type=str.lower, help="...")`?
then args.minimal_build is [optional parameters] (meaning minimal build with optional config), or None (meaning no minimal build). and we can check for exact string match of the parameters